libs/sdk-ui-pivot/src/impl/privateTypes.ts

```diff
@@ -21,6 +21,7 @@ export interface IMenuAggregationClickConfig {
     measureIdentifiers: string[];
     attributeIdentifier: string;
     include: boolean;
+    isRowOrColumn?: string;
```

Comment:
Can we define better type than just string? The naming isRowOrColumn sounds for me like boolean (is prefix)